### PR TITLE
[WIP] logs: initial

### PR DESCRIPTION
**Pull request by pared**
Matches iterative/dvc/pull/4902

### CODE REVIEW

**Comment by jorgeorpinel** [comment]
Hi. This is the related core PR, right? iterative/dvc/pull/4902

File: content/docs/command-reference/run.md [hunk 1]
```diff
@@ -5,6 +5,8 @@ command.
 
 ## Synopsis
 
+# TODO logs
+
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:9]: Is this a hypothetical section to explain logs? Would be an H3 under **Description**, prob 🙂

File: content/docs/command-reference/run.md [hunk 2]
```diff
@@ -221,6 +223,22 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
   up to the user to manage them separately. See also the difference between `-o`
   and `-O`.
 
+- `--logs <path>` - specify a directory that will contain `dvclive` logs (#TODO
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:226]: What's `dvclive`?

File: content/docs/command-reference/run.md [hunk 3]
```diff
@@ -221,6 +223,22 @@ $ dvc run -n my_stage './my_script.sh $MYENVVAR'
...
+  link) compatible structure:
+
+  ```<path>
+     ├── history
+     │   ├── m1.tsv
+     │   └── m2.tsv
+     └── latest.json
+  ```
+
+  It is a convinience option that tells dvc that under `<path>/history` are
```
Comment (jorgeorpinel) [inline content/docs/command-reference/run.md:236]:
```suggestion
  It is a convinience option that tells DVC that under `<path>/history` are
```

**Comment by pared** [comment]
@jorgeorpinel, yes this is issue related o iterative/dvc#4902. Currently it does not represent what should be included, for now its a reminder for me to write it up, once we are sure how logs will be named and how they will behave.

**Comment by jorgeorpinel** [comment]
OK I resolved everything. Please let me know when to help here.

**Comment by pared** [comment]
Closing in favor of #2227

**Pull request closed without merging**